packages/sandcastle/templates/Sandcastle.ts

```diff
@@ -150,15 +157,19 @@ const Sandcastle = {
    */
   addToolbarButton(text: string, onclick: () => void, toolbarId?: string) {
     Sandcastle.declare(onclick);
+
     const button = document.createElement("button");
     button.type = "button";
-    button.className = "cesium-button";
+    button.className = "stratakit-mimic-button";
+    button.dataset.kiwiVariant = "solid";
```

Comment:
Same comment here and in `addToolbarMenu`.

"kiwi" is non-obvious to those who don't know the previous name of stratakit. I think that warrant a comment to explain.